Source/relay/TourGuide/Items of the Month/2021/Daylight Shavings Helmet.ash

```diff
@@ -1,31 +1,44 @@
 //Daylight Shavings Helmet
 
 string [int, string] dshBuffOrder = {
-	{"Spectacle": "+50% item and +5 " + HTMLGenerateSpanFont("Spooky", "grey") + " resist"},
-	{"Toiletbrush": "+25 ML and +5 " + HTMLGenerateSpanFont("Stench", "green") + " resist"},
-	{"Barbell": "+25 Muscle and +50% Gear drops"},
-	{"Grizzly": "+25-50 MP regen and +5 " + HTMLGenerateSpanFont("Cold", "blue") + " resist"},
-	{"Surrealist": "+25 Mysticality and +50% Food drops"},
-	{"Musician": "+25 Moxie and +50% Booze drops"},
-	{"Gull-Wing": "+100% init and +5 " + HTMLGenerateSpanFont("Hot", "red") + " resist"},
-	{"Warlord": "+25 Weapon damage and +10% Crit"},
-	{"Wizard": "+25 Spell damage and +10% Spell Crit"},
-	{"Cowboy": "+25 Ranged damage and +50 maximum HP"},
-	{"Friendly": "+100% meat and +5 " + HTMLGenerateSpanFont("Sleaze", "purple") + " resist"}
+	{"Spectacle Moustache": "+50% item and +5 " + HTMLGenerateSpanFont("Spooky", "grey") + " resist"},
+	{"Toiletbrush Moustache": "+25 ML and +5 " + HTMLGenerateSpanFont("Stench", "green") + " resist"},
+	{"Barbell Moustache": "+25 Muscle and +50% Gear drops"},
+	{"Grizzly Beard": "+25-50 MP regen and +5 " + HTMLGenerateSpanFont("Cold", "blue") + " resist"},
+	{"Surrealist's Moustache": "+25 Mysticality and +50% Food drops"},
+	{"Musician's Musician's Moustache": "+25 Moxie and +50% Booze drops"},
+	{"Gull-Wing Moustache": "+100% init and +5 " + HTMLGenerateSpanFont("Hot", "red") + " resist"},
+	{"Space Warlord's Beard": "+25 Weapon damage and +10% Crit"},
+	{"Pointy Wizard Beard": "+25 Spell damage and +10% Spell Crit"},
+	{"Cowboy Stache": "+25 Ranged damage and +50 maximum HP"},
+	{"Friendly Chops": "+100% meat and +5 " + HTMLGenerateSpanFont("Sleaze", "purple") + " resist"}
```

Comment:
Used these full names to get easy indexing based on the actual buff name. If we want to keep the shortened names, we can add another attribute to this map for the display name in the table – let me know if you have a strong preference for that.

Reply:
nah, long names are fine, i think this is more readable. thanks fo noting it tho.